lib/data.rb

```diff
@@ -20,7 +21,7 @@ def self.define(*args, &block)
         Data.const_set(name, klass)
       end
 
-      klass.define_singleton_method(:members) { args.map{ _1.intern } }
```

Comment:
I don't know if this `intern` was important, but it didn't fail any tests to remove it.